src/store/holochain.js

```diff
@@ -213,6 +224,11 @@ export default {
         }
       }, 1000)
     },
+    signalHoloDisconnect ({ commit }) {
+      if (!isHoloHosted()) return log('Attemtped to signal Holo as disconnected when not in Holo Context')
+        commit('setReconnecting', 0)
+        commit('setIsChaperoneDisconnected', true)
```

Comment:
```suggestion
      if (!isHoloHosted()) {
        log('Attemtped to signal Holo as disconnected when not in Holo Context')
        return
      }
      commit('setReconnecting', 0)
      commit('setIsChaperoneDisconnected', true)
```

Same as above, I think returning a value we don't care about makes the code harder to read

Reply:
Also, do we need this log? Seems like this is not unexpected behavior outside of a holo context, and the log doesn't really tell us anything useful in that context.

Reply:
Updated without log.